library/bash

```diff
@@ -1,49 +1,54 @@
-# this file is generated via https://github.com/tianon/docker-bash/blob/96e8d89c9f89a0d22f04af7584c64b7e2a74087b/generate-stackbrew-library.sh
+# this file is generated via https://github.com/tianon/docker-bash/blob/a9c27dc3ab0927afc456dfe6032682bf6ba2992e/generate-stackbrew-library.sh
 
 Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon)
 GitRepo: https://github.com/tianon/docker-bash.git
 
+Tags: devel-20190705, devel
```

Comment:
Copying my note from https://github.com/tianon/docker-bash/commit/11b071d:

> Chet updates a "devel" branch in https://git.savannah.gnu.org/cgit/bash.git roughly every seven days, which is a very reasonable cadence and worth publishing an image for.

Reply:
Given that upstream specifically creates these ~weekly snapshots, that seems reasonable. I don't want to create images where we arbitrarily choose a snapshot commit.